src/main/java/net/atos/zac/zaaksturing/ZaakafhandelParameterBeheerService.java

```diff
@@ -55,16 +53,10 @@ public class ZaakafhandelParameterBeheerService {
     @Inject
     private ZTCClientService ztcClientService;
 
-    public ZaakafhandelParameters createZaakafhandelParameters(final ZaakafhandelParameters zaakafhandelParameters) {
-        valideerObject(zaakafhandelParameters);
-        zaakafhandelParameters.getHumanTaskParametersCollection().forEach(ValidationUtil::valideerObject);
-        zaakafhandelParameters.getUserEventListenerParametersCollection().forEach(ValidationUtil::valideerObject);
-        zaakafhandelParameters.setCreatiedatum(ZonedDateTime.now());
-        entityManager.persist(zaakafhandelParameters);
-        return zaakafhandelParameters;
-    }
+    @Inject
+    private ZaakafhandelParameterService zaakafhandelParameterService;
```

Comment:
De ZaakafhandelParameterBeheerService gebruikt de zaakafhandelParameterService en de zaakafhandelParameterService gebruikt weer de ZaakafhandelParameterBeheerService.
Hiermee creeer je dus een loop qua afhankelijkheid. Is dat echt nodig? Ik zou niet verwachten dat deze servicdes elkaar uberhaupt nodig hebben. 
Creeer anders een derde (helper) bean waar de  ZaakafhandelParameterBeheerService en ZaakafhandelParameterService gebruik van maken.

Verzin een list :-)

Reply:
heb de afhankelijkheid van beheer -> service weggehaald